recipes/xgeo/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "1.0" %}
+
+package:
+  name: xgeo
+  version: {{ version }}
+
+source:
+  url: https://github.com/Geosynopsis/xgeo/archive/xgeo-v{{ version }}.tar.gz
+  sha256: 2b494a56c8f91967828c49806a0e8cdde2f4af8c24bc2b2ede509ce6ff96a7dc
+
+build:
+  number: 0
+  script:  "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [(py<36)]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas>=0.24.1
+    - xarray>=0.11.3
+    - rasterio>=1.0.18
+    - geopandas>=0.4.0
+    - netCDF4>=1.4.2
+    - shapely>=1.6.4
+
+test:
+  imports:
+    - xarray
+    - xgeo
+
+about:
+  home: https://geosynopsis.github.io/xgeo
+  license: Apache 2.0
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: Apache-2.0
```